lib/apiservers/engine/backends/container.go

```diff
@@ -257,7 +258,8 @@ func (c *Container) TaskWaitToStart(cid, cname, eid string) error {
 
 // ContainerExecCreate sets up an exec in a running container.
 func (c *Container) ContainerExecCreate(name string, config *types.ExecConfig) (string, error) {
-	defer trace.End(trace.Begin(name))
+	op := trace.NewOperation(context.TODO(), "")
+	defer trace.End(trace.Begin(op.Sprintf("name=(%s)", name)))
```

Comment:
op.Sprintf -> fmt.Sprintf

Reply:
wait why? I added the op.Sprintf since i thought that was the `stringer` that you wanted :)

Reply:
ah I just saw the below

Reply:
hmmm 👍 I can make that change, I see what you meant by stringer now, I still like the Sprintf though unless there is an egregious reason not to have it.

Reply:
Personally  I don't think having a Sprintf on op makes sense. fmt already provides that and that's the standart way of doing that.

Others can chime in if they think this way or that way :)